android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -165,21 +165,30 @@ class ReactExoplayerView extends FrameLayout implements
     private final AudioManager audioManager;
     private final AudioBecomingNoisyReceiver audioBecomingNoisyReceiver;
 
+    // store last progress event values to avoid sending unnecessary messages
+    private long lastPos = -1;
+    private long lastBufferDuration = -1;
+    private long lastDuration = -1;
+
     private final Handler progressHandler = new Handler() {
         @Override
         public void handleMessage(Message msg) {
             switch (msg.what) {
                 case SHOW_PROGRESS:
-                    if (player != null
```

Comment:
Why did you remove this null check? Wouldn't it be safer to leave it there? (I'm OK with removing the state checks, but null check might be beneficial here).

Reply:
I fully agree ! change done